Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -27,6 +28,8 @@ internal class InMemoryRepository<TItem> : IRepository<TItem> where TItem : clas
         internal long CurrentTs => DateTimeOffset.UtcNow.ToUnixTimeSeconds();
         internal ConcurrentDictionary<string, string> Items { get; } = new();
 
+        internal EventHandler<ChangeFeedItemArgs<TItem>> Changes { get; set; }
```

Comment:
Could this just be an `Action<T>` instead an `EventHandler<T>`?

Reply:
We could if we are sure there are no more subscribers to this than the one we have currently, does it hurt to leave it as it?

Reply:
An `Action<T>` is also a multicast delegate, it just doesn't add the `object` parameter to the method signature.

Reply:
Ah, that is cool to know, learn something new everyday!